troubleshooting.rst

```diff
@@ -41,6 +42,28 @@ File picking doesn't work or crashes
 You need to install the :code:`zenity` utility:
 - Ubuntu - :code:`sudo apt-get install zenity`
 
+Game starts with Qt UI, but not with regular launcher
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Qt UI stores the game it's own directory per version in :code:`~/.local/share/mcpelauncher/versions.
+
+:code:`mcpelauncher-client` expects :code:`libs` and :code:`assets` directories in :code:`~/.local/share/mpelauncher` per default.
+
+You have two options:
+ - Copy, move or simlink those directories of your desired version to that location.
+ - Start :code:`mcpelauncher-client` with :code:`-dg ~/.local/share/mcpelauncher/versions/DESIRED_VERSION`.
+
+ I cannot find my worlds
```

Comment:
I suggest creating a separete FAQ page and put the two pieces of information in there.

Reply:
Done. I decided against an FAQ since Troubleshooting already has a lot of useful information, and 2 or 3 FAQ items wouldn't justify one. But I guess once started, it can always expand.